src/main/webapp/resources/js/pages/admin/components/AdminUserGroupsPage.jsx

```diff
@@ -19,8 +19,8 @@ export default function AdminUserGroupsPage() {
   return (
       <RolesProvider rolesFn={getUserGroupRoles}>
         <Router>
-          <UserGroupsPage baseUrl={"/admin/groups"} path={setBaseUrl("")} />
-          <UserGroupsDetailsPage baseUrl={"/admin/groups"} path={setBaseUrl("/:id")} />
+          <UserGroupsPage baseUrl={setBaseUrl("/admin/groups")} path={setBaseUrl("")} />
```

Comment:
Since `setBaseUrl("/admin/groups")` is called multiple times it would be better to set this as a constant